### PR TITLE
Fixes #326 added 'visit' to launch message

### LINKED ISSUE #326: Launch messaging
When running the project from Xcode the console just says
```
[1464327605] [INFO] Server starting …
```
whereas `vapor run` will print
```
Running...
Visit http://localhost:8080
[1464328127] [INFO] Server starting ...
[1464328133] [INFO] GET /
[1464328133] [INFO] GET /styles/app.css
[1464328133] [INFO] GET /images/vapor-logo.png
```

Should roll the that messaging into the same place (the first line `Runnin…` is from `vapor-cli`, not sure where `Visit…` is from yet) and make it clearer when the server is actually done starting up.

### PR DESCRIPTION
**Pull request by feinstruktur**

### CODE REVIEW

**Comment by tannernelson** [comment]
If we add this here, we should make sure to remove it from the Example project or we'll get double the messages.

**Comment by feinstruktur** [comment]
Yes, that makes sense. I'll make a reminder to do so but it won't be before Monday most likely.

File: Sources/Vapor/Core/Application.swift [hunk 1]
```diff
@@ -167,7 +167,8 @@ extension Application {
 
     private func startServer() {
         do {
-            Log.info("Server starting ...")
+            Log.info("Server starting...")
+            Log.info("Visit http://\(host):\(port)")
```
Comment (LoganWright) [inline Sources/Vapor/Core/Application.swift]: Should this go after server initialization? Or should we condense to just one message.

Comment (feinstruktur) [inline Sources/Vapor/Core/Application.swift]: I'd have preferred to stick it after actual server start up as well but `try server.start()` is blocking and going deeper is not an option as there can be any number of `Server` types :/

I'll roll it into one then.

**Comment by tannernelson** [comment]
The way the application starts has been refactored in a different PR. I put this change in there. I should have merged this one first, sorry >_<

**Pull request closed without merging**